src/planner/Admin.h

```diff
@@ -57,16 +57,21 @@ class DropNode : public SingleDependencyNode {
     bool ifExist_{false};
 };
 
-class ShowHosts final : public SingleDependencyNode {
+class ShowHosts final : public SingleInputNode {
```

Comment:
Why change it to `SingleInput`?

Reply:
`SingleDependencyNode ` is the right one.

Reply:
good catch, fixed.